controllers/Users/userController.js

```diff
@@ -0,0 +1,39 @@
+const User = require('../../lib/modelManagers/usermodel');
+
+const pagination = require('../../lib/utils/pagination/paginationHelper');
+
+/**
+ *
+ *
+ * @class Products
+ */
+class UsersController {
+  /**
+   * @static
+   * @returns { object } object
```

Comment:
` * @returns { object } object` should be  `@returns undefined`  because nothing is being returned from this function.

Reply:
fixed